spec/features/admin_company_management_spec.rb

```diff
@@ -44,12 +44,26 @@ def login
 
       expect(page).to have_button("Delete Company")
       click_button("Delete Company")
-
       expect(current_path).to eq(admin_dashboard_index_path)
-
       expect(Company.count).to eq(4)
-
     end
 
+    it "can add notes to a contacted company" do
+      companies = create_list(:company, 5, status: "contacted")
+      login
+
+      within(".contacted-companies") do
+        click_link(companies.first.organization, match: :first)
+      end
+      within(".add_note") do
+        fill_in "note", with: "test note"
+      end
+      click_link_or_button("Add Note")
+
+      within(".notes") do
+        expect(page).to have_content("test note")
+      end
+      expect(current_path).to eq(admin_company_path(companies.last.id))
+    end
   end
 end
```

Comment:
I agree.  However, the user story only involved adding notes.  We can add another user story, but I think that should be done in another PR.